src/services/lnd/utils.ts

```diff
@@ -425,7 +424,7 @@ export const validate = async ({
     } catch (err) {
       const error = `Error decoding the invoice`
       logger.error({ params, success: false, error }, error)
-      throw new LoggedError(error)
+      return new UnknownLnInvoiceDecodeError(error)
```

Comment:
This error should only originate from the [`ln-invoice` decoder](https://github.com/GaloyMoney/galoy/blob/main/src/domain/bitcoin/lightning/ln-invoice.ts). It appears this function duplicates the well typed function [here](https://github.com/GaloyMoney/galoy/blob/main/src/domain/bitcoin/lightning/ln-invoice.ts#L16) - can you switch the whole thing?

Reply:
not straightforward, the rest of the function in this file expect different property than what `decodeInvoice` return. I have added a TODO and should be done in another PR.